src/script/lua_api/l_object.cpp

```diff
@@ -580,6 +580,29 @@ int ObjectRef::l_get_eye_offset(lua_State *L)
 	return 2;
 }
 
+// send_mapblock(self, pos)
+int ObjectRef::l_send_mapblock(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	ObjectRef *ref = checkobject(L, 1);
+	check_v3s16(L, 2);
+
+	// player
+	RemotePlayer *player = getplayer(ref);
+	if (!player)
+		return 0;
+	// pos
+	v3s16 p = read_v3s16(L, 2);
+
+	// Do it
```

Comment:
useless com

Reply:
This comment is in all the other functions in this file, too.

Reply:
I'll nuke it though, it annoys me too :)